### ISSUE TITLE
Create dev tools image for CHE Dev Spaces for UI testing

### ISSUE THREAD

**Issue by musienko-maxim** [eclipse/che#22293]
### Is your enhancement related to a problem? Please describe

During the DevTools week, I tried to use the https://github.com/openshift/console project (frontend part). In this repo, we have UI tests based on the Cypress framework. After changing some limits the build project operations. navigation to code, autocompletion worked pretty comfortably. But the main problem is the run UI test in the workspace because the UDI image does not contain Xvfb and browser. There is make impossible the tests form the DevSpace

### Describe the solution you'd like

As a variant, we have Cypress's official images: https://github.com/cypress-io/cypress-docker-images. Potentially it can be changed/adapted for devtools container and we can be able to run UI tests

**Comment by musienko-maxim** [comment]
On the first iteration of DevTools I fixed problems with the resource limits. I made it possible to run DevWorkspace with a console Cypress project and fetch node dependencies, working with autocomplete, open/find files, and other features. However, this project is supposed to use UI tests and interactions with a browser. So the next challenge - try to modify UDI image add the X-libs and Google Chrome browser and try to launch a Cypress test.

**Comment by musienko-maxim** [comment]
So, we decided to change tactics and try running UI tests from the [e2e tests repository of Eclipse Che](https://github.com/eclipse/che/tree/main/tests/e2e).

However, as described above, the issue with modifying the UDI image remains unresolved. I experimented with exec commands inside the image, but it was the wrong approach. We need to take a closer look at the article: [CDE Customization by Mario Loriedo](https://che.eclipseprojects.io/2024/02/05/@mario.loriedo-cde-customization.html). It's crucial to study it more deeply and attempt the process again.
